src/index.js

```diff
@@ -4,6 +4,7 @@ let app = require('./server').default;
 
 if (module.hot) {
   module.hot.accept('./server', () => {
+    // eslint-disable-next-line no-console
```

Comment:
Add an issue to add a logger like the one we have for the API. Then we won't get any of these errors.